components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -18,52 +18,427 @@ module omero {
+                 *
+                 * @param experimenter A new {@link omero.model.Experimenter}
+                 *        instance. Not null.
+                 * @param defaultGroup Instance of
+                 *        {@link omero.model.ExperimenterGroup}. Not null.
+                 * @param otherGroups Array of
+                 *        {@link omero.model.ExperimenterGroup} instances. Can
+                 *        be null.
+                 * @return id of the newly created
+                 *         {@link omero.model.Experimenter} Not null.
+                 */
                 long createExperimenter(omero::model::Experimenter user,
                                         omero::model::ExperimenterGroup defaultGroup, ExperimenterGroupList groups) throws ServerError;
+
+                /**
+                 * Creates and returns a new user in the given groups with
+                 * password.
+                 *
+                 * @param experimenter A new {@link omero.model.Experimenter}
+                 *        instance. Not null.
+                 * @param password Not-null. Must pass validation in the
+                 *        security sub-system.
+                 * @param defaultGroup Instance of
+                 *        {@link omero.model.ExperimenterGroup}. Not null.
+                 * @param otherGroups Array of
+                 *        {@link omero.model.ExperimenterGroup} instances. Can
+                 *        be null.
+                 * @return id of the newly created
+                 *         {@link omero.model.Experimenter} Not null.
+                 * @throws SecurityViolation if the new password is too weak.
+                 */
                 long createExperimenterWithPassword(omero::model::Experimenter user, omero::RString password,
                                                     omero::model::ExperimenterGroup defaultGroup, ExperimenterGroupList groups) throws ServerError;
+
+                /**
+                 * Creates and returns a new group. The
+                 * {@link omero.model.Details#setPermissions} method should be
+                 * called on the instance which is passed. The given
+                 * {@link omero.model.Permissions} will become the default for
+                 * all objects created while logged into this group, possibly
+                 * modified by the user's umask settings.
+                 * If no permissions is set, the default will be
+                 * {@link omero.model.Permissions#USER_PRIVATE},
+                 * i.e. a group in which no user can see the other group
+                 * member's data.
+                 *
+                 * @param group  a new
+                 * {@link omero.model.ExperimenterGroup} instance. Not null.
+                 * @return id of the newly created {@link ExperimenterGroup}
+                 * @see <a href="https://trac.openmicroscopy.org/ome/ticket/1434">ticket:1434"</a>
```

Comment:
I know this is a copy across from ome but do you want to pick up minor fixes here? If so then there is a stray double quote:

ticket:1434" -> ticket:1434